### PR TITLE
[#42] [iOS] [UI] As a user, I can see the thank you screen

### LINKED ISSUE #42: [iOS] [UI] As a user, I can see the thank you screen
## Why

Show Thank you screen to notify user that their survey is completed.

## Acceptance Criteria

- Show the icon.
- Show the title "Thanks for taking  the survey."

## Resources

Screen Shot 2565-09-12 at 11 49 49

[Thanks for taking the survey](https://www.figma.com/file/GjRPOjDyZ6f4EDL3wKarRK/Challenge---Mobile-App?node-id=36%3A28)

### PR DESCRIPTION
**Pull request by blyscuit**
- close #42 

## What happened

Add Thank you screen UI.

## Insight

- Add `Lottie-ios` pod.
- Add `LottieView` to convert Lottie to SwiftUI.
- Use `Moko` for Colors.
- Add `ColorResource` extension for SwiftUI.
 
## Proof Of Work

https://user-images.githubusercontent.com/6356137/219267644-34bb4a5a-0d00-4d3a-881f-c447b6e9cda2.mp4

### CODE REVIEW

File: shared/src/commonMain/resources/MR/base/colors.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- format: #RRGGBB[AA] or 0xRRGGBB[AA] or RRGGBB[AA] where [AA] - optional -->
+    <color name="backgroundColor">#15151A</color>
+    <color name="white">#FFFFFF</color>
```
Comment (minhnimble) [inline shared/src/commonMain/resources/MR/base/colors.xml]: Do we really need this color declaration - `white`? can't we access the default white color per platform? 🤔

File: iosApp/Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift [hunk 2]
```diff
@@ -26,10 +27,21 @@ struct SurveyDetailView: View {
     // TODO: Replace with real answer object
     @State var currentAnswers = [String]()
     @State var isShowingQuitPrompt = false
+    @State var isShowingSuccess = false
```
Comment (minhnimble) [inline iosApp/Survey/Sources/Presentation/Modules/SurveyDetail/SurveyDetailView.swift]: Optional: small naming suggestion for better clarity 🙏
```suggestion
    @State var isShowingSuccessConfirmation = false
```

**Review by suho — APPROVED** [review]

File: shared/src/commonMain/resources/MR/base/colors.xml (hunk 1 above)
Comment (blyscuit) [reply shared/src/commonMain/resources/MR/base/colors.xml]: Updated this name to `primary` for clarity.

**Review by minhnimble — APPROVED** [review]
LGTM 🚀

**Pull request merged**